packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2015,18 +2113,18 @@ export class ContainerRuntime
 	 * Parse an op's type and actual content from given serialized content
 	 * ! Note: this format needs to be in-line with what is set in the "ContainerRuntime.submit(...)" method
 	 */
-	private parseOpContent(serializedContent?: string): ContainerRuntimeMessage {
-		assert(serializedContent !== undefined, 0x6d5 /* content must be defined */);
-		const { type, contents, compatDetails }: ContainerRuntimeMessage =
-			JSON.parse(serializedContent);
-		assert(type !== undefined, 0x6d6 /* incorrect op content format */);
-		return { type, contents, compatDetails };
+	private parseLocalOpContent(serializedContents?: string): LocalContainerRuntimeMessage {
```

Comment:
Should be `Outbound` not `Local`, I'm pretty sure.  See comment thread below at the top of `applyStashedOp`